docs/docs/hypermedia-protocol/document-blocks.md

```diff
@@ -117,10 +117,12 @@ One of:
 
 - [`Heading`](#heading-block)
 - [`Paragraph`](#paragraph-block)
+- [`Code Block`](#code-block)
+- [`Math Block`](#math-block)
```

Comment:
A math block and code block are not "text blocks" in my opinion because the text annotations are not used